### ISSUE TITLE
Shadow cljs can't find needed files

### ISSUE THREAD

**Issue by JovanToroman** [thheller/shadow-cljs#714]
Hi, with the newest version of shadow-cljs (shadow-cljs@2.9.3) I get an error:
```
Execution error (FileNotFoundException) at clojure.main/main (main.java:40).
Could not locate shadow/cljs/cli__init.class, shadow/cljs/cli.clj or shadow/cljs/cli.cljc on classpath.
```
The error happens when I try compiling a clojurescript app. It seems that it is the latest shadow-cljs that introduced the error.

**Comment by thheller** [comment]
This is likely because you are using npm shadow-cljs version with the wrong real shadow-cljs version. Ie. npm shadow-cljs@2.9.3 but not 2.9.3 in project.clj/deps.edn

**Comment by crofty** [comment]
Thanks for logging this.  I just got bitten by it too and had no idea what was going on.  Updating the version in my deps.edn fixed it.

**Comment by thheller** [comment]
I just released `2.9.4` which reverts the change so it uses the old namespace again. If have the npm version `2.9.3` you should upgrade as it is only compatible with 2.9.3+ but not older versions. 2.9.4 is once again compatible with older versions.
